aten/src/ATen/native/native_functions.yaml

```diff
@@ -4087,7 +4087,6 @@
     CUDA: batch_norm_cuda
     MPS: batch_norm_mps
     MkldnnCPU: mkldnn_batch_norm
-  tags: core
```

Comment:
cc @SherlockNoMad - confirming that we're removing `native_batch_norm` from core aten decomps (since we can't functionalize it)